[Term Entry] JavaScript Dates .getUTCMonth() #2426 (#2477)
Codecademy/docs: content/javascript/concepts/dates/terms/getUTCMonth/getUTCMonth.md

---

> Title:

Reviewer: Please review another entry or template and update the metadata to match. There are several edits that are required:

- all of the content must be within single quotes
- the title and description should not begin on a separate line
- catalogcontent should include two items

‘.getUTCMonth()'
Description:

> ‘Method of date instances that returns the month date according to universal time, as a zero-based value (where zero indicates the first month of the year).' 

Reviewer: ```suggestion
'Returns the month of the passed date according to universal time, as a zero-based value.' 
```

Subjects: 
  - ‘Web Development’
  - ‘Computer Science’
Tags:
  - ‘Date’
  - 'Methods’
CatalogContent:
 - 'introduction-to-javascript'
---

> get methods retrieve the date components based on the user’s local timezone settings. For increased control over the dates and times, you can use the **`.getUTCMonth()`** method, which are exactly the same as the `getMonth()` method, except they calculate the time based on the UTC (Coordinated Universal Time) standard.

Reviewer: ```suggestion
Get methods are used to retrieve date components based on the user’s local timezone settings. The **`.getUTCMonth()`** method returns the month of the date passed, similar to `.getMonth()`. However, `.getUTCMonth()` calculates the time based on the UTC (Coordinated Universal Time) standard.
```

> ## Syntax

Reviewer: add a line break after the header

> ```

Reviewer: ```suggestion
```pseudo
```

const today = new Date();
const month = today.getUTCMonth();

> ```

Reviewer: add a line below the syntax

> Example above assigns the month portion of the current `Date` to the variable `month`.

Reviewer: update this line to explain the function, the return value, etc in more detail.

> ## Example

Reviewer: add an empty line after the header

```js
const date1 = new Date('December 31, 1975, 23:15:30 GMT+11:00');
const date2 = new Date('December 31, 1975, 23:15:30 GMT-11:00');

console.log(date1.getUTCMonth());

> // expected output December: 11

Reviewer: delete the comments and use a shell block instead
